app/helpers/hubstats/metrics_helper.rb

```diff
@@ -0,0 +1,47 @@
+module Hubstats
+  module MetricsHelper
+    # include ActionView::Helpers::ApplicationHelper
+    # Public - Shows statistics for all of the data Hubstats has within the date range
+    #
+    # Returns - the stats for the entirety of Hubstats
+    def dashboard
```

Comment:
instead of assigning a bunch of local variables in this method you could break each of them up into their own method.

You wouldn't even have to change 
```ruby
      @stats_row_one = {
        developer_count: developer_count,
        pull_count: pull_count,
        pulls_per_dev: pulls_per_dev,
        deploy_count: deploy_count,
        net_additions: net_additions
      }
      @stats_row_two = {
        reviewer_count: reviewer_count,
        comment_count: comment_count,
        comments_per_rev: comments_per_rev,
        avg_additions: additions.round.to_i,
        avg_deletions: deletions.round.to_i
      }
```

This is ok for now but the method is a bit long and does lots of stuff.

One thing I really try to do with ruby methods is make them as short as possible and only give them one responsibility.

The responsibility of this method is 
* get user counts
* get reviewer counts
* get deploy counts
* get pull request counts
* get comment counts
* calculate reviews
* calculate pr per dev
* get adds
* get deletions
* format data into usable data structure

I think all of those responsibilities should be their own thing and the dashboard method should only do the last one.
You can put all these methods right into the helper if you want but once you do that it's much easier to change how we get that data later without changing too much stuff at once.